jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -310,7 +315,16 @@ public void reconnectionSuccessful()
                 }
             }
 
-            @Override
+            /**
+             * The connection will retry to reconnect in the specified number of seconds.
+             * <p>
+             * Note: This method is only called if {@link ReconnectionManager#isAutomaticReconnectEnabled()} returns true, i.e.
+             * only when the reconnection manager is enabled for the connection.
+             * </p>
+             *
+             * @param i remaining seconds before attempting a reconnection.
+             */
+            @Deprecated
```

Comment:
This method has been removed from smack. Since it contains some functionality I've only marked it as deprecated.
Was that correct? Or should we keep the method regardless of what smack did?